homework_1.html

```diff
@@ -9,17 +9,36 @@
         <div class = "wrapper">
              <div class = "header">TODO List</div>
              <div class = "sidebar_left">
-                 <p>Pure JavaScript</p>
+                 <span id = "padd">Pure JavaScript</span>
                  <div class = "input_but">
```

Comment:
- Better to use full words for classes and variables (`input-button` instead of `input-but`)